CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ BUG FIXES:
 * Fix `vcd_independent_disk` reapply issue, which was seen when optional `bus_sub_type` and `bus_type` wasn't used - [GH-394]
 * Fix `vcd_vapp_network` apply issue, where the property `guest_vlan_allowed` was applied only to the last of multiple networks.
 * `datasource/vcd_network_direct` is now readable by Org User (previously it was only by Sys Admin), as this change made it possible to get the details of External Network as Org User [GH-408]
+* vCD 9.0 - TBD
```

Comment:
Something missing here

Reply:
Yes indeed. This was "something to make an empty commit initially - thanks. Removed.